From 75ad1345d4af5268448a9b26797294b48795cd69 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?K=C3=A9vin=20Petit?= <kevin.petit@arm.com>
Date: Thu, 29 Feb 2024 17:46:38 +0000
Subject: [PATCH] Remove redundant function declarations from source/operand.h
 (#5584)

Flagged by -Wredundant-decls

I'm assuming the declarations in libspirv.h are part of the external
interface and need to be kept.


Change-Id: I6b138d3322a7a4ee49ee33b0fbcf0ca35dd92261

Signed-off-by: Kevin Petit <kevin.petit@arm.com>
---
 source/operand.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/source/operand.h b/source/operand.h
index a3010d9341..f74c93389e 100644
--- a/source/operand.h
+++ b/source/operand.h
@@ -57,12 +57,6 @@ spv_result_t spvOperandTableValueLookup(spv_target_env,
 // Gets the name string of the non-variable operand type.
 const char* spvOperandTypeStr(spv_operand_type_t type);
 
-// Returns true if the given type is concrete.
-bool spvOperandIsConcrete(spv_operand_type_t type);
-
-// Returns true if the given type is concrete and also a mask.
-bool spvOperandIsConcreteMask(spv_operand_type_t type);
-
 // Returns true if an operand of the given type is optional.
 bool spvOperandIsOptional(spv_operand_type_t type);
 
